packages/core/src/command/patient-import/patient-import-local.ts

```diff
@@ -0,0 +1,143 @@
+import { executeAsynchronously } from "../../util/concurrency";
+import { createJobRecord } from "./commands/create-job-record";
+import { validateAndParsePatientImportCsvFromS3 } from "./commands/validate-and-parse-import";
+import { checkPatientRecord } from "./commands/check-patient-record";
+import { creatOrUpdatePatientRecord } from "./commands/create-or-update-patient-record";
+import { startDocumentQuery } from "./commands/start-document-query";
+import { createPatient } from "./commands/create-patient";
+import { startPatientQuery } from "./commands/start-patient-query";
+import {
+  PatientImportHandler,
+  StartPatientImportRequest,
+  ProcessPatientImportRequest,
+  ProcessPatientCreateRequest,
+  ProcessPatientQueryRequest,
+} from "./patient-import";
+import { createPatientPayload } from "./patient-import-shared";
+
+export class PatientImportHandlerLocal implements PatientImportHandler {
+  constructor(private readonly patientImportBucket: string) {}
+
+  async startPatientImport({
+    cxId,
+    facilityId,
+    jobId,
+    rerunPdOnNewDemographics = true,
+    dryrun = false,
+  }: StartPatientImportRequest): Promise<void> {
+    const jobStartedAt = new Date().toISOString();
+    const processPatientImportRequest: ProcessPatientImportRequest = {
+      cxId,
+      facilityId,
+      jobId,
+      jobStartedAt,
+      s3BucketName: this.patientImportBucket,
+      processPatientCreateQueue: "local",
+      rerunPdOnNewDemographics,
+      dryrun,
+    };
+    const boundProcessPatientImport = this.processPatientImport.bind(this);
+    await boundProcessPatientImport(processPatientImportRequest);
```

Comment:
For later (assuming this works): why this and not just `await this.boundProcessPatientImport(...)`?

Reply:
Because the calls are chained, this loses context without binding it each. Bind returns a function. To make it one line you'd have `await  this.processPatientImport.bind(this)(processPatientImportRequest);` which I thought looked strange :).

Reply:
I don't understand the issue, let's talk about this on the next PRs. 👍